src/pcm_plugin.c

```diff
@@ -67,6 +67,7 @@ enum {
     PCM_PLUG_STATE_SETUP,
     PCM_PLUG_STATE_PREPARED,
     PCM_PLUG_STATE_RUNNING,
+    PCM_PLUG_STATE_DRAINING,
```

Comment:
what is the purpose of this state

Reply:
Currently, this is not used anywhere. But STATE_DRAINING is defined at pcm: https://github.com/tinyalsa/tinyalsa/blob/4fbaeef03cd1cb216e0f356c0433ca70f8b9c464/include/tinyalsa/pcm.h#L127
I thought it's more complete to define this in plugin as well.
What's your advice? Should I remove this?

Reply:
I don't think there is a need to add this state unless we are using it for handling some scenarios

Reply:
Thanks. Accept and update.